src/components/smart_objects/include/smart_objects/object_schema_item.h

```diff
@@ -50,6 +50,11 @@ namespace ns_smart_objects {
  * @brief Object member.
  **/
 struct SMember {
+  /*
+   *  @brief Defines data type that SMember contains.
+   */
+  enum class Type { OEM_SPECIFIC = 0, RPC_SPECIFIC = 1 };
```

Comment:
```suggestion
  enum class Type { POLICY = 0, API = 1 };
```

Reply:
Might not be needed, since I proposed a solution which doesn't need this enum

Reply:
@jacobkeeler fixed in 986bcf0